src/index.md

```diff
@@ -1,13 +1,25 @@
 # Welcome to Deis Workflow
 
-Deis (pronounced DAY-iss) Workflow is an open source Platform as a Service (PaaS) that adds a developer-friendly layer to any Kubernetes cluster, making it easy to deploy and manage applications on your own servers.
+Deis (pronounced DAY-iss) Workflow is an open source Platform as a Service (PaaS) that adds a developer-friendly layer
+to any Kubernetes cluster, making it easy to deploy and manage applications.
 
-Workflow includes capabilities for building and deploying from source via `git push`, configuring your applications, creating and rolling back releases, managing application domain names and SSL certificates, providing edge routing, aggregating application logs, and controlling access to application resources.  All of this is exposed through a simple REST API and command line interface.
+Deis Workflow includes capabilities for building and deploying from source via `git push`, configuring your applications,
+creating and rolling back releases, managing application domain names and SSL certificates, providing edge routing,
+aggregating application logs, and controlling access to application resources. All of this is exposed through a simple
+REST API and command line interface.
 
-For more detailed information about Workflow, view the [Concepts][concepts] and [Architecture][arch] sections.
+## Getting Started
+
+Take a deep dive into Deis Workflow in our [Concepts][concepts], [Architecture][arch], and [Components][components]
```

Comment:
I've always agreed with this sentiment. +1 for communicating docs in third person.